### PR TITLE
Updating test cases

### PR DESCRIPTION
**Pull request by dragan-n-dream**

### CODE REVIEW

File: js/screen/app/partials/_main_view.html [hunk 1]
```diff
@@ -81,6 +83,31 @@ <h2 class="page-header">Controller</h2>
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="page-header">Saving player state</h2>
```
Comment (marc-n-dream) [inline js/screen/app/partials/_main_view.html:88]: This is the wrong heading, isn't it?

```suggestion
      <h2 class="page-header">Localisation</h2>
```

File: js/screen/app/ctrls/main_ctrl.js [hunk 2]
```diff
@@ -31,53 +65,67 @@ TestApp.controllers.controller('MainCtrl', ['$scope', '$location', '$timeout', f
         checked: false
       }
     ],
+    localisation:[
+      {
+        text: "I tested that the game loads the game in the same language as the main screen (browser or Android TV). If the game does not support the language of the screen, the game defaults to English. NOTE: Only the language of the screen is relevant to the game, not the language of the controllers.",
```
Comment (marc-n-dream) [inline js/screen/app/ctrls/main_ctrl.js:70]: Is the note actually necessary? The developer can not decide what language to get the localised data in, that is decided by the AirConsole API before the screen / controller loads.

If it is necessary, the Game Localisation section should point to the localisation guide to ensure that developers use the correct API.

File: js/screen/app/partials/_main_view.html [hunk 3]
```diff
@@ -81,6 +83,31 @@ <h2 class="page-header">Controller</h2>
...
+            <input type="checkbox" ng-checked="q.checked" ng-model="questions.localisation[$index].checked" ng-change="updateProgress(q)"> <span ng-bind-html="q.text"></span>
+          </label>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="page-header">Localisation</h2>
```
Comment (marc-n-dream) [inline js/screen/app/partials/_main_view.html]: This is the wrong heading, isn't it?

```suggestion
      <h2 class="page-header">Player state persistence</h2>
```

**Comment by marc-n-dream** [comment]
A few pointers added in the PR.

I think it would be good to prefix all cases that start with **I have tested** with a case number like `QC-1:  I have tested my game with unstable connection and higher latency.`
Did the Content team have an opinion on this?

**Review by marc-n-dream — CHANGES_REQUESTED** [review]

File: js/screen/app/partials/_main_view.html (hunk 3 above)
Comment (dragan-n-dream) [reply js/screen/app/partials/_main_view.html]: ✅

File: js/screen/app/partials/_main_view.html (hunk 1 above)
Comment (dragan-n-dream) [reply js/screen/app/partials/_main_view.html:88]: ✅

**Pull request merged**